freqtrade/strategy/backtest_lookahead_bias_checker.py

```diff
@@ -0,0 +1,226 @@
+import copy
+from copy import deepcopy
+from datetime import datetime, timedelta, timezone
+
+import pandas
```

Comment:
i think it'd be better to import just the dataframe (`from pandas import DataFrame`).

if we "must" import the whole pandas library - then please use the pandas convention of `as pd` ... :)

Reply:
done

Reply:
done